### ISSUE TITLE
Add removal of `FromHex` from all hashes to Changelog.

### ISSUE THREAD

**Issue by sanket1729** [rust-bitcoin/rust-bitcoin#1747]
We should add a note in changelog that these were replaced by corresponding `from_str` definitions. Overall, I like like that we don't have to import the trait to parse from strings.

**Comment by Aankirz** [comment]
Hey @sanket1729 , I would like to solve this issue I'm a beginner to this repo but have understood about rust bitcoin and what it does, 

Can you tell me more about this issue, pls

I've to make changes in hashes/CHANGELOG.md right ?
Here's a approach
Like this : 0.X.0 - 2023-04-08 (Where X is the release number)
### Removed
- Removed `FromHex` trait from all hashes.
  - Parsing from strings is now done using the corresponding `FromStr` implementations.

**Comment by Aankirz** [comment]
@apoelstra , Thanks for suggesting the changes, but I need a little help in understanding about this issue,

As suggested by you, I also need to mention the PR link, so do I need to replace FromHex with FromStr in the impl code ?


Thanks for helping

**Comment by apoelstra** [comment]
@Aankirz no, I cannot help you. I encourage you to take time to learn git and learn about interacting with open-source projects, then come back and try again!

**Comment by Aankirz** [comment]
Ok that's completely fine, thanks for the suggestions
